Services/OperationService/Controllers/OperationsController.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using DTO;
+using DTO.MarketBrokerObjects;
+using System.Threading.Tasks;
+using Interfaces;
+using DTO.BrokerRequests;
+
+namespace OperationService.Controllers
+{
+    [ApiController]
+    [Route("[controller]")]
+    public class OperationsController : ControllerBase
+    {
+        [Route("instruments/get")]
+        [HttpGet]
+        public async Task<IEnumerable<Instrument>> GetInstruments(
+                [FromServices] ICommand<GetInstrumentsRequest, IEnumerable<Instrument>> command,
+                [FromQuery] BrokerType broker,
```

Comment:
Why do we use from query here? Isn't it would be easier to achieve some model from body here?

Reply:
> Why do we use from query here? Isn't it would be easier to achieve some model from body here?

Because it's GET